src/stories/button/Button.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {
+  StyledButtonComponent,
+  OutlineButtons,
+  TextButtons,
+} from './Button.style';
+
+const Button = ({ color, type, text, icon }) => {
+  if (type === 'contained') {
+    return (
+      <StyledButtonComponent color={color}>
+        {icon} {text}
+      </StyledButtonComponent>
+    );
+  }
+
+  if (type === 'outlined') {
+    return (
+      <OutlineButtons color={color}>
+        {icon} {text}
+      </OutlineButtons>
+    );
+  }
+  if (type === 'text') {
+    return <TextButtons color={color}>{text}</TextButtons>;
+  }
+
+  return null;
```

Comment:
@Arbina001  you return direct this instead  of null and remove if condition`return <TextButtons color={color}>{text}</TextButtons>;`